docs/core/tools/global-tools.md

```diff
@@ -1,135 +1,288 @@
+Entry is added to the manifest file /Users/name/tools/sub/dotnet-tools.json
+```
+
+The following example shows a manifest file with two local tools installed:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "botsay": {
+      "version": "1.0.0",
+      "commands": [
+        "botsay"
+      ]
+    },
+    "dotnetsay": {
+      "version": "2.1.3",
+      "commands": [
+        "dotnetsay"
+      ]
+    }
+  }
+}
+```
+
+You typically add a local tool to the root directory of the repository. After you check in the manifest file to the repository, developers who check out code from the repository get the latest manifest file. To install all of the tools listed in the manifest file, they run the `dotnet tool restore` command:
+
+```dotnetcli
+dotnet tool restore
+```
 
-If the command is `dotnetsay`, you call it with:
+The output indicates which tools were restored:
+
+```console
+Tool 'botsay' (version '1.0.0') was restored. Available commands: botsay
+Tool 'dotnetsay' (version '2.1.3') was restored. Available commands: dotnetsay
+Restore was successful.
+```
+
+## Install a specific tool version
+
+To install a pre-release version or a specific version of a tool, specify the version number by using the `--version` option, as shown in the following example:
+
+```dotnetcli
+dotnet tool install dotnetsay --version 2.1.3
+```
+
+## Use a tool
+
+The command that you use to invoke a tool may be different from the name of the package that you install. To display all of the tools currently installed on the machine for the current user, use the [dotnet tool list](dotnet-tool-list.md) command:
+
+```dotnetcli
+dotnet tool list
+```
+
+The output shows each tool's version and command, similar to the following example:
+
+```console
+Package Id      Version      Commands       Manifest
+------------------------------------------------------------------------------------------------------
+botsay          1.0.0        botsay         C:\Users\username\Documents\botsay\.config\dotnet-tools.json
+dotnetsay       2.1.3        dotnetsay      C:\Users\username\Documents\botsay\.config\dotnet-tools.json
+```
+
+As shown in this example, the list shows local tools. To see global tools, use the `--global` option, and to see tool-path global tools, use the `--tool-path` option.
+
+### Invoke a global tool
+
+For global tools, use the tool command by itself. For example:
+
+If the command is `dotnetsay`, that's what you use to invoke the command:
 
 ```console
 dotnetsay
 ```
 
-If the tool author wanted the tool to appear in the context of the `dotnet` prompt, they may have written it in a way that you call it as `dotnet <command>`, such as:
+If the tool author specified a command with the prefix `dotnet-`, you can invoke the tool with or without the `dotnet` command. For example, if the command is `dotnet-doc`, you can invoke the tool in either of the following ways:
 
 ```dotnetcli
+dotnet-doc
 dotnet doc
```

Comment:
You can invoke it with dotnet-doc without using dotnet in front of it.

Reply:
Perhaps we should add a note here and/or in the "invoke a local tool" section that the dotnet- prefix is mandatory for local tools but not global tools, just to be perfectly clear. It seems like an arbitrary rule, so might be worth clarifying.

Reply:
Also, the example keeps confusing me because it seems to contradict with the text above it. Is it that the dotnet- prefix isn't required, or that the dotnet command isn't required?